lib/rubocop/cop/style/documentation_method.rb

```diff
@@ -0,0 +1,76 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for missing top-level documentation of
+      # methods. Private methods are excluded from this cop
+      class DocumentationMethod < Cop
+        include AnnotationComment
+
+        MSG = 'Missing top-level %s documentation method comment.'.freeze
```

Comment:
I don't quite get this message. I'd go for "Missing documentation comment for public method.".